astrology.lic

```diff
@@ -22,6 +22,7 @@ class Astrology
     # Make sure that our Cirlce is up-to-date - avoid using stale data for new characters who joined the guild
     bput('info', 'Circle:') if DRStats.circle.zero?
 
+    @equipmentmanager = EquipmentManager.new
```

Comment:
Personally, I'd prefer `@equipment_manager`, but it's not a big deal

Reply:
I can change that